debian/rules

```diff
@@ -30,3 +30,8 @@ override_dh_install:
 	mv debian/celestia-common/usr/share/celestia/celestia.cfg \
 	   debian/celestia-common/etc/celestia.cfg
 	ln -s /etc/celestia.cfg debian/celestia-common/usr/share/celestia/
+	rm debian/celestia-common/usr/share/celestia/fonts/*
+	ln -s /usr/share/fonts/truetype/dejavu/DejaVuSans.ttf \
+	      debian/celestia-common/usr/share/celestia/fonts/
+	ln -s /usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf \
```

Comment:
in celestia.cfg it reads `DejaVuSans-Bold.ttf`
https://github.com/CelestiaProject/Celestia/blob/master/celestia.cfg#L226

Reply:
sharp eye :)

I copy-pasted wrong name :)